src/components/BillCard/BillCard.svelte

```diff
@@ -5,16 +5,16 @@
 	import type { BillStatus } from '$lib/politigraph/genql';
 	import Proposer from '$components/Proposer/Proposer.svelte';
 	import type { ComponentProps } from 'svelte';
+	import dayjs from 'dayjs';
 
 	export let id: string;
 	export let nickname: string | null;
 	export let title: string | null;
 	export let proposedOn: Date | null;
+	export let startedOn: Date | null = null;
 	export let status: BillStatus;
 	export let proposer: ComponentProps<Proposer>['proposer'] = undefined;
-	export let orientation: 'landscape' | 'portrait' = 'landscape';
-	export let currentState: string | undefined = undefined;
-	export let daySinceProposed: number | undefined = undefined;
+	export let orientation: 'landscape' | 'portrait' = 'portrait';
```

Comment:
You change the props default value, which is a breaking change. If you want to change this then you need to search all usage of the `BillCard` in the repo and add/remove the orientation props to make sure the behavior after the props change will be the same. (Including the one that was commented out)

Reply:
Yup sorry for that but since we are in the process of redesigning backend database (Politigraph) some function is not ready and was commented out, thise code will returned when the data is ready.
- `daysSinceProposal` is not necessary anymore cause you calculate it with other props. You can remove it.
- `current` is also not used anyway. You can remove it too. If someday we need it we can bring it back.
So current changes is fine.